app/templates/macros/toolkit_forms.html

```diff
@@ -0,0 +1,144 @@
+{% macro text(question_content, service_data, errors, question_number=None) -%}
```

Comment:
Hey they are but we don't have a neat way of sharing them at the moment and given they don't contain a huge amount of the logic I've decided to leave them for the moment.